app/src/androidTest/java/com/github/epfl/meili/photo/CameraActivityTest.kt

```diff
@@ -69,7 +69,9 @@ class CameraActivityTest {
     @Test
     fun takePhotoWithVolumeDownTest() {
         val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+        Thread.sleep(1000)
         mDevice.pressKeyCode(KeyEvent.KEYCODE_VOLUME_DOWN)
+        Thread.sleep(1000)
```

Comment:
Why are these sleeps being added, is the CameraActivity slower now?

Reply:
They are added so that the tests pass.
This is a test so it has no impact on the camera activity.

Reply:
What I meant was that the tests were passing without this before, so the fact that we need to add this now implies that the camera activity is slower and we need to wait more for the activity to get ready. Is that correct?

AFAIK the problem is in the photo edit tests, not the camera ones.

Reply:
They weren't always passing on cirrus